### PR TITLE
Feature/154: Update Student Verification Field Name

### PR DESCRIPTION
**Pull request by kunaal-gupta**

### CODE REVIEW

**Comment by dimassudjito** [comment]
Please use "develop" branch as pull request target next time instead of "main" branch

**Review by MoyinF — CHANGES_REQUESTED** [review]
Integrate the develop branch and fix any possible conflicts.

**Review by yismailuofa — APPROVED** [review]
Same as before LGTM, but please add a justification / background and title in your PR.

File: mobile/.expo/settings.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "hostType": "tunnel",
+  "hostType": "lan",
```
Comment (ArmaanKatyal) [inline mobile/.expo/settings.json]: We don't want to make any changes to the frontend. So can you please change it to its original value.

**Review by ArmaanKatyal — CHANGES_REQUESTED** [review]
Can you please remove all the .idea files from the pull request and update your .gitignore. 
Other than that, please test if the changes made in the model are reflected in the admin page and the MongoDB database.

File: mobile/.expo/settings.json (hunk 1 above)
Comment (kunaal-gupta) [reply mobile/.expo/settings.json]: How do we test changes in the admin page & MangoDB?

**Review by yismailuofa — CHANGES_REQUESTED** [review]
Can we remove all the .idea files (excluding the .gitignore)

Also, @dimassudjito should we also remove desktop.ini and package-lock.json?

**Comment by dimassudjito** [comment]
> Can we remove all the .idea files (excluding the .gitignore)
> 
> Also, @dimassudjito should we also remove desktop.ini and package-lock.json?

@yismailuofa 

I think we should remove all changes unrelated to the ticket. It's probably easier to create a new branch and just copy the change since we just changed one file (Student.js). It'll probably take 2 mins if you just do it hahah

**Comment by yismailuofa** [comment]
@kunaal-gupta 
Leaving this up to you, could either remove all the files besides Student.js on this PR,
Or make a new one and request a review from me I'll approve it

**Comment by yismailuofa** [comment]
@dimassudjito I removed the extra files myself. Can you take a look and merge if all is good?

**Pull request merged**